### PR TITLE
Add Button type (2)

### PR DESCRIPTION
**Pull request by roderickdevelopers**
Please look these over and let me know if they are all correct. I committed each file separately in case I needed a change on one of them.

### CODE REVIEW

File: guides/release/components/template-lifecycle-dom-and-modifiers.md [hunk 1]
```diff
@@ -317,18 +317,18 @@ Let's start with the HTML we're working with:
...
 Next, let's add an event handler to the `Play` button:
 
 ```handlebars {data-filename="app/components/audio-player.hbs" data-diff="-3,+4"}
 <audio src={{@srcURL}} />
 
-<button>Play</button>
-<button {{on "click" this.play}}>Play</button>
-<button>Pause</button>
+<button type="button">Play</button>
+<button type="button"> {{on "click" this.play}}>Play</button>
```
Comment (ijlee2) [inline guides/release/components/template-lifecycle-dom-and-modifiers.md:330]:
```suggestion
<button type="button" {{on "click" this.play}}>Play</button>
```

File: guides/release/components/template-lifecycle-dom-and-modifiers.md [hunk 2]
```diff
@@ -190,7 +190,7 @@ export default class CounterComponent extends Component {
 ```handlebars {data-filename="app/components/counter.hbs"}
 <p>{{this.count}}</p>
 
-<button {{on "click" this.increment}}>+</button>
+<button type="button"> {{on "click" this.increment}}>+</button>
```
Comment (ijlee2) [inline guides/release/components/template-lifecycle-dom-and-modifiers.md:193]:
```suggestion
<button type="button" {{on "click" this.increment}}>+</button>
```

File: guides/release/components/template-lifecycle-dom-and-modifiers.md [hunk 3]
```diff
@@ -355,8 +355,8 @@ In this case, we'll assign the `<audio>` element to the `audioElement` property
 <audio src={{@srcURL}} />
 <audio src={{@srcURL}} {{ref this "audioElement"}} />
 
-<button {{on "click" this.play}}>Play</button>
-<button>Pause</button>
+<button type="button"> {{on "click" this.play}}>Play</button>
```
Comment (ijlee2) [inline guides/release/components/template-lifecycle-dom-and-modifiers.md:358]:
```suggestion
<button type="button" {{on "click" this.play}}>Play</button>
```

**Comment by ijlee2** [comment]
@roderickdevelopers It looks like you used find and replace. Can you look at all examples that have the `{{on}}` modifier again?

**Comment by roderickdevelopers** [comment]
@ijlee2 I actually didn't use find and replace so that is my mistake and I will fix it ASAP

**Comment by roderickdevelopers** [comment]
Pushed another commit to fix @ijlee2 suggestion for the {{on}} modifiers.

**Comment by ijlee2** [comment]
Looks good to me! I don't think I have permissions to approve Percy diff and merge PR, so I'll let someone review your code too.

**Comment by roderickdevelopers** [comment]
@amyrlam Ok great suggestions. Yes another pull request tomorrow and I think that will finish up release folder.

**Pull request merged**